src/queries.ts

```diff
@@ -41,18 +41,22 @@ export const submissionKeys = {
 
 export function addDefaultMutationFns(queryClient: QueryClient) {
   queryClient.setMutationDefaults(submissionKeys.details(), {
-    mutationFn: async (updated: SubmissionMetadata) => {
+    mutationFn: async (updated: SubmissionMetadataUpdate) => {
```

Comment:
Side note (**not** a deal breaker for this PR): I think I'd prefer a different variable name from `updated`; e.g. `updatedSubmissionMetadata` or `updatedSM` or something. This is not the focus of this PR/these changes—I just wanted to express that thought.